frontend/src/components/main/main.js

```diff
@@ -18,16 +18,17 @@ function MainPage() {
                     <div className="recentGames">
                         <h1 className="recentGamesHeader">Recent Games: </h1>   
                         <Game />
-                        <Game />  
-                        <Game />  
-                        <Game />   
+                        <Game />
```

Comment:
I'm guessing you could have an express route for '/latestGames' and you can get them here and pass the data as props into the Game components?

Reply:
Yeah I think that's the plan!